<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_estoque</class>
 <widget class="QDialog" name="tela_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource_file.qrc">
    <normaloff>:/logo cin.png</normaloff>:/logo cin.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(230, 230, 230);
color:black;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab_widget">
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
    border: 1px solid #ccc; /* Border around the tab widget */
    background-color: #f0f0f0; /* Background color of the tab content area */
}

QTabBar::tab {
    background: rgb(255, 150, 150); /* Background color of inactive tabs */
    border: 1px solid #ccc; /* Border around tabs */
    padding: 10px; /* Padding inside tabs */
    font-size: 14px; /* Font size of tab text */
    color: #333; /* Text color of inactive tabs */
}

QTabBar::tab:selected {
    background: rgb(255, 170, 170); /* Slightly lighter/different shade for the active tab */
    border: 1px solid #333; /* Wider border for the selected tab */
    border-bottom-color: rgb(255, 170, 170); /* Match the border color to the background */
    color: #000; /* Text color of the selected tab */
}

QTabBar::tab:hover {
    background: rgb(255, 120, 120); /* Slightly darker red when hovered */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* Add a small margin to non-selected tabs */
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_nova">
      <attribute name="title">
       <string>Novo Produto</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_codigo">
               <property name="text">
                <string>Código</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_codigo">
               <property name="styleSheet">
                <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_descricao">
               <property name="text">
                <string>Descrição</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_descricao">
               <property name="styleSheet">
                <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_qtd">
               <property name="text">
                <string>Quantidade estoque</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_qtd">
               <property name="styleSheet">
                <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_valor">
               <property name="text">
                <string>Valor</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_valor">
               <property name="styleSheet">
                <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_novo">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
             </property>
             <property name="text">
              <string>Novo</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_gerar_novo">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
             </property>
             <property name="text">
              <string>Cadastrar</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_gestao">
      <attribute name="title">
       <string>Gestão de Estoque</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_codigo_ge">
             <property name="text">
              <string>Código</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_codigo_ge">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="label_descricao_ge">
             <property name="text">
              <string>Descrição</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_descricao_ge">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="label_qtd_ge">
             <property name="text">
              <string>Quantidade Estoque</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_qtd_ge">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="label_valor_ge">
             <property name="text">
              <string>Valor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_valor_ge">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_att_ge">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
             </property>
             <property name="text">
              <string>Atualizar</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_excluir_ge">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
             </property>
             <property name="text">
              <string>Excluir</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QRadioButton" name="radioButton_codigo">
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
    width: 10px;
    height: 10px;
    border: 1px solid black;
    border-radius: 5px;
    background-color: #f0f0f0;
}

QRadioButton::indicator:checked {
    background-color: rgb(255, 150, 150);
    border: 1px solid black;
}

</string>
             </property>
             <property name="text">
              <string>Código</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_descricao">
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
    width: 10px;
    height: 10px;
    border: 1px solid black;
    border-radius: 5px;
    background-color: #f0f0f0;
}

QRadioButton::indicator:checked {
    background-color: rgb(255, 150, 150);
    border: 1px solid black;
}

</string>
             </property>
             <property name="text">
              <string>Descrição</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLineEdit" name="lineEdit_procurar">
             <property name="styleSheet">
              <string notr="true">background-color: white;
color:black;
border: 1px solid black;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_procurar">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
             </property>
             <property name="text">
              <string>Pesquisar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="tableWidget_produtos">
         <property name="styleSheet">
          <string notr="true">QTableWidget {
    background-color: white;
    color: black;
    gridline-color: #e1e1e1;
	border: 2px solid black
}

QTableWidget::item:selected {
    background-color: #ffcccc; 
    color: black;
}

QTableWidget::horizontalHeader {
    background-color: #f0f0f0;
    border: none;
}

QTableWidget::verticalHeader {
    background-color: #f0f0f0;
    border: none;
}

QTableWidget::item {
    padding: 5px;
}
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource_file.qrc"/>
 </resources>
 <connections/>
</ui>
